### PR TITLE
[rename] ignore docusaurus and dockerhub, but go after others.

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

Remove more references to libra where possible.

### Have you read the [Contributing Guidelines on pull requests]

Yes

## Test Plan

combo of CI for gha and @JoelMarcey for the website changes.

## Related PRs

None

### CODE REVIEW

**Comment by JoelMarcey** [comment]
@rexhoffman Would also like to land this after the actual site switch. And we can do whatever rebasing/merging after that.

File: developers.diem.com/src/README.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ unzip diem.zip
 ## Publishing
 
 The site is hosted on GitHub pages, via the `gh-pages` branch of the `website'
-[GitHub repo](https://github.com/libra/website).
+[GitHub repo](https://github.com/diem/website).
```
Comment (JoelMarcey) [inline developers.diem.com/src/README.md:48]: None of this publishing stuff is true any longer. I think we need a vet of the READMEs and other info files for accuracy. We can leave for now and do a cleanup pass later.

File: developers.diem.com/docs/node/overview.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ Learn how to start, configure, and monitor a Diem Node
     to="/docs/node/nodes#validator-nodes-validators"
   />
   <SimpleTextCard
-    icon="img/libra-nodes.svg"
-    iconDark="img/libra-nodes-dark.svg"
+    icon="img/diem-nodes.svg"
```
Comment (JoelMarcey) [inline developers.diem.com/docs/node/overview.md:20]: We would need to rename the actual images too, right? 

They would be in `developers.diem.com/static/img`

**Review by JoelMarcey — CHANGES_REQUESTED** [review]
Need to rename a couple of actual image files since you change their name here.

I would also rebase this too and repush.

**Comment by rexhoffman** [comment]
@JoelMarcey sure this can wait, are we getting this remamed?   

https://github.com/dfuzr/libra-developer-plugins

Referenced heavily in the site.

**Comment by libra-action** [comment]
**release binaries dependency change summary**:
```
target packages:
  M ureq 1.5.4 (direct third-party, crates.io)
    * version upgraded from 1.5.2
    * (unchanged features: json, native-tls, serde, serde_json)
  M libc 0.2.81 (transitive third-party, crates.io)
    * added features: align
    * (unchanged features: default, std)
  M lock_api 0.4.2 (transitive third-party, crates.io)
    * version upgraded from 0.4.1
    * (unchanged features: [none])
  M mio 0.6.23 (transitive third-party, crates.io)
    * version upgraded from 0.6.22
    * (unchanged features: default, with-deprecated)
  M openssl-sys 0.9.59 (transitive third-party, crates.io)
    * version upgraded from 0.9.58
    * (unchanged features: [none])
  M parking_lot 0.11.1 (transitive third-party, crates.io)
    * version upgraded from 0.11.0
    * (unchanged features: default)
  M socket2 0.3.18 (transitive third-party, crates.io)
    * version upgraded from 0.3.15
    * (unchanged features: [none])
  M tracing 0.1.22 (transitive third-party, crates.io)
    * version upgraded from 0.1.21
    * (unchanged features: log, std)
```

**Comment by libra-action** [comment]
**LSR dependency change summary**:
```
target packages:
  M ureq 1.5.4 (direct third-party, crates.io)
    * version upgraded from 1.5.2
    * (unchanged features: json, native-tls, serde, serde_json)
  M lock_api 0.4.2 (transitive third-party, crates.io)
    * version upgraded from 0.4.1
    * (unchanged features: [none])
  M openssl-sys 0.9.59 (transitive third-party, crates.io)
    * version upgraded from 0.9.58
    * (unchanged features: [none])
  M parking_lot 0.11.1 (transitive third-party, crates.io)
    * version upgraded from 0.11.0
    * (unchanged features: default)
```

**Comment by libra-action** [comment]
**LEC dependency change summary**:
```
target packages:
  M ureq 1.5.4 (direct third-party, crates.io)
    * version upgraded from 1.5.2
    * (unchanged features: json, native-tls, serde, serde_json)
  M libc 0.2.81 (transitive third-party, crates.io)
    * added features: align
    * (unchanged features: default, std)
  M lock_api 0.4.2 (transitive third-party, crates.io)
    * version upgraded from 0.4.1
    * (unchanged features: [none])
  M mio 0.6.23 (transitive third-party, crates.io)
    * version upgraded from 0.6.22
    * (unchanged features: default, with-deprecated)
  M openssl-sys 0.9.59 (transitive third-party, crates.io)
    * version upgraded from 0.9.58
    * (unchanged features: [none])
  M parking_lot 0.11.1 (transitive third-party, crates.io)
    * version upgraded from 0.11.0
    * (unchanged features: default)
  M socket2 0.3.18 (transitive third-party, crates.io)
    * version upgraded from 0.3.15
    * (unchanged features: [none])
  M tracing 0.1.22 (transitive third-party, crates.io)
    * version upgraded from 0.1.21
    * (unchanged features: log, std)
```

**Pull request closed without merging**